src/main/java/uk/gov/hmcts/reform/sscs/service/ServiceHearingsService.java

```diff
@@ -63,6 +63,15 @@ public ServiceHearingValues getServiceHearingValues(ServiceHearingRequest reques
         return model;
     }
 
+    private ServiceHearingValues getModel(SscsCaseData caseData) throws ListingException, UpdateCaseException {
```

Comment:
Could we make this method name a bit less generic, getServiceHearingValuesModel or something like that?

Reply:
done